services/supabase/migrations/20260827000001_restrict_project_visibility_by_status.sql

```
@@ -0,0 +1,32 @@
+-- Restrict public project visibility to active/paused/funded campaigns.
+-- Draft and review projects remain accessible to:
+--   • Platform admins (role = 'admin')
+--   • The project owner (kindler_id = auth.uid())
+--   • Authorized project members (core / admin / editor roles)
+
+-- Drop the existing permissive public-read policy
+DROP POLICY IF EXISTS "Projects are viewable by everyone" ON projects;
```

SQLFluff (4.3.0)

[error] Do not use special characters in identifiers.

(RF05)